panel/tests/test_param.py

```diff
@@ -25,6 +25,10 @@
 )
 
 
+def Pane(obj, **kwargs):
+    return PaneBase.get_pane_type(obj, **kwargs)(obj, **kwargs)
```

Comment:
What's up with this? Not easy to replace with `pn.panel`?

Reply:
It is for functions that expect a `pn.Param` output.  `pn.panel` give a `Column` output. 

I could also change the type to assert with to `Column` if this is more desirable for you. 

``` python
def test_instantiate_from_class():

    class Test(param.Parameterized):

        a = param.Number()

    assert isinstance(Pane(Test), Param)
```

Reply:
Gotcha